### PR TITLE
Update Process documentation with ProcessBuilder removal

### PR DESCRIPTION
**Pull request by pyrech**

### CODE REVIEW

File: components/process.rst [hunk 1]
```diff
@@ -258,37 +258,13 @@ instead::
...
-:class:`Symfony\\Component\\Process\\ProcessBuilder` class instead::
+To make your code work better on all platforms, you might want to pass an array of
+arguments to the constructor instead::
 
-    use Symfony\Component\Process\ProcessBuilder;
-
-    $builder = new ProcessBuilder(array('ls', '-lsa'));
-    $builder->getProcess()->run();
-
-In case you are building a binary driver, you can use the
```
Comment (pyrech) [inline components/process.rst]: Not sure about this removal but it looks like we can no longer register a prefix

**Comment by javiereguiluz** [comment]
Thanks for this ... but sadly I must close it as a duplicate of #8879 which was recently merged.

**Comment by pyrech** [comment]
No problem :+1: I opened the PR without looking to the existing ones, sorry.

The related PR is fine but remove the documentation about passing an array of arguments. Is that wanted?

**Comment by javiereguiluz** [comment]
You are right! I forgot to mention that we have another PR to document this feature: #8881

**Comment by pyrech** [comment]
Then it's all good! Thanks

**Pull request closed without merging**